.storybook/webpack.config.js

```diff
@@ -21,30 +22,31 @@ module.exports = async ({ config }) => {
         {
             test: /\.tsx?$/,
             exclude: /node_modules/,
-            use: [
-                { loader: 'babel-loader' },
-                { loader: 'react-docgen-typescript-loader' }
-            ]
+            use: [{ loader: 'babel-loader' }, { loader: 'react-docgen-typescript-loader' }],
+        },
+        {
+            test: /\.(js?|ts?)$/,
+            exclude: /node_modules\/(?!(box-annotations)\/).*/,
+            loader: 'source-map-loader',
+            enforce: 'pre',
         },
     );
     if (shouldIncludeAllSupportedBrowsers) {
-        config.module.rules.push(
-            {
-                test: /\.(js|ts|tsx)$/,
-                include: [
-                    /node_modules\/@hapi\/address/,
-                    /node_modules\/react-intl/,
-                    /node_modules\/intl-messageformat-parser/,
-                    /node_modules\/intl-messageformat/,
-                ],
-                loader: 'babel-loader',
-                options: {
-                    babelrc: false,
-                    compact: false,
-                    configFile: path.resolve('babel.config.js'),
-                },
-            }
-        );
+        config.module.rules.push({
```

Comment:
Did this get run through prettier or something? Seems weird that the formatting would change.

Reply:
Yeah